tests/test_security_gating.py

```
@@ -0,0 +1,615 @@
+async def test_auth_routes_log_only_size_and_type(client, caplog):
+    """Auth routes (/api/auth/, /api/login, /api/register, /api/logout) log only size/type, not content."""
+    auth_endpoints = ["/api/login", "/api/register", "/api/logout", "/api/auth/refresh"]
+    sensitive_data = {"username": "test", "password": "secret123", "token": "abc"}
+
+    for endpoint in auth_endpoints:
+        caplog.clear()
+        with caplog.at_level(logging.WARNING):
+            try:
+                _ = await client.post(endpoint, json=sensitive_data)
+            except Exception:
+                pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)